src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
+ ** `sudo`: similar to `none` regarding the connection to the head node, but a `sudo` command will be initiated to impersonate as the current Scheduler User, before doing a job submission.
+ * `alternateRMUrl` : the url used by the ProActive Nodes to contact <<_glossary_resource_manager,*ProActive Resource Manager*>>. This url is displayed on ProActive server startup. Example: `pnp://myserver:64738`.
+ * `sshPort` : port used for ssh connections.
+ * `nsPreCommand` : a linux command which can be run before launching ProActive Nodes on the cluster. Can be used as a workaround when some system environment variables are not properly set when starting ProActive Nodes.
+ * `nsSubmitCommand` : this is the main command used to start ProActive Nodes on the cluster. The command can used patterns which will be replaced dynamically by the ProActive Resource Manager. +
+[cols=2*]
+|===
+
+|`%NS_JOBNAME%` |contains a configurable job name dynamically created by the resource manager.
+
+|`%LOG_FILE%` |contains a log file path dynamically created by the resource manager and located in side the NSSchedulerHome installation. This log file is useful to debug errors during native scheduler job submission.
+
+|`%PA_USERNAME%` |contains the current ProActive Scheduler user.
+|===
+
+Depending on the actual native scheduler implementation, `nsSubmitCommand` will vary, here are examples definitions: +
+
+[cols=2*]
+|===
+
+|PBS |`qsub -N %NS_JOBNAME% -o %LOG_FILE% -j oe`
+
+|SLURM |`sbatch -J %NS_JOBNAME% -o %LOG_FILE%`
+
+|LSF |`bsub -J %NS_JOBNAME% -o %LOG_FILE% -e %LOG_FILE%`
+
+|===
+
+ * `nsKillCommand` : this is the command used to kill ProActive Nodes started previously by the nsSubmitCommand. It can use the following patterns:
+
+[cols=2*]
+|===
+
+|`%NS_JOBNAME%` |contains a configurable job name dynamically created by the resource manager.
+
+|`%NS_JOBID%` |contains the job id returned by the native scheduler when submitting the job. Currently, job id can only be used with PBS, when the setting `submitReturnsJobId` is set to `true`.
+
+|===
+
+Similarly to nsSubmitCommand, `nsKillCommand` will vary for each native scheduler syntax: +
+
+[cols=2*]
+|===
+
+|PBS |`qdel %NS_JOBID%`
+
+|SLURM |`scancel -n %NS_JOBNAME%`
+
+|LSF |`bkill -J %NS_JOBNAME%`
+
+|===
+
+ * `submitReturnsJobId`: is the job id returned plainly when calling the nsSubmitCommand? This is the behavior of PBS, and this is why this setting should be set to `true` when using PBS.
+ * `nsJobName`: a way to configure the `%NS_JOBNAME%` pattern. The following patterns can be used:
+
+[cols=2*]
+|===
+
+|`%PA_TASKID%` |contains the ProActive Task and Job ID associated with the node request.
+
+|`%PA_USERNAME%` |contains the current ProActive Scheduler user.
+
+|===
+
+ * `maxDeploymentFailure`: NativeSchedulerInfrastructure can make several attempts when starting a ProActive Node on the cluster.
```

Comment:
We can just say `The number of attempts...`.

Reply:
ok